tests/test_wires.py

```diff
@@ -218,36 +245,5 @@ def test_unique_wires_method(self):
         res = Wires.unique_wires([wires2, wires1, wires3])
         assert res == Wires([3, 1])
 
-        with pytest.raises(WireError, match="expected a `pennylane.wires.Wires` object"):
+        with pytest.raises(WireError, match="Expected a `Wires` object"):
             Wires.unique_wires([[2, 1], [8, 5]])
-
-    def test_merge_method(self):
```

Comment:
How come this test was deleted? From what I saw, the method is still there :thinking:

Reply:
Method should have been deleted!